distribution/docs/src/main/resources/_contents/_configuring/configuring-catalog-provider-contents.adoc

```diff
@@ -0,0 +1,30 @@
+
+==== Reconfiguring ${branding} with a Different Catalog Provider
```

Comment:
The dummy provider doesn't exist anymore. Maybe we should instead have two sub-sections, one to explaining how to switch to the Solr External provider and one for a non-Solr external one?